src/org/smssecure/smssecure/sms/IncomingTextMessage.java

```diff
@@ -34,6 +34,7 @@ public IncomingTextMessage createFromParcel(Parcel in) {
   private final String  groupId;
   private final boolean push;
   private final int     subscriptionId;
+  private final boolean receivedWhenLocked;
 
   public IncomingTextMessage(SmsMessage message, int subscriptionId) {
```

Comment:
Shouldn't this just be: `this(message, subscriptionId, false)` now that the other constructor exists?